Examples/ArcGISToolkitExamples/ARExample.swift

```diff
@@ -0,0 +1,102 @@
+import UIKit
+import ARKit
+import ArcGISToolkit
+import ArcGIS
+
+open class ARExample: UIViewController {
+    
+    public let arView = ArcGISARView(frame: .zero)
+
+    override open func viewDidLoad() {
+        super.viewDidLoad()
+        
+        // Example of how to get ARSessionDelegate methods from the ArcGISARView.
+        arView.sessionDelegate = self
+        arView.arSCNViewDelegate = self
+
+        view.addSubview(arView)
+        arView.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            arView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            arView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            arView.topAnchor.constraint(equalTo: view.topAnchor),
+            arView.bottomAnchor.constraint(equalTo: view.bottomAnchor)
+            ])
+        
+        arView.sceneView.scene = makeStreetsScene()
+    }
+    
+    override open func viewDidAppear(_ animated: Bool) {
+        arView.startTracking()
+    }
+    
+    override open func viewDidDisappear(_ animated: Bool) {
+        arView.stopTracking()
+    }
+
+    private func makeStreetsScene() -> AGSScene {
+        
+        // create scene with the streets basemap
+        let scene = AGSScene(basemapType: .streets)
+        
+        // create elevation surface
+        let elevationSource = AGSArcGISTiledElevationSource(url: URL(string: "http://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer")!)
+        let surface = AGSSurface()
+        surface.elevationSources = [elevationSource]
+        surface.name = "baseSurface"
+        surface.isEnabled = true
+        surface.backgroundGrid.isVisible = false
+        scene.baseSurface = surface
+        
+        return scene
+    }
+}
+
+extension ARExample: ARSessionDelegate {
+    
+    public func session(_ session: ARSession, didUpdate frame: ARFrame) {
+        // Example of how to get ARSessionDelegate methods from the ArcGISARView.
+    }
+}
+
+extension ARExample: ARSCNViewDelegate {
+    
+    public func session(_ session: ARSession, didFailWithError error: Error) {
+        guard error is ARError else { return }
+        
+        let errorWithInfo = error as NSError
+        let messages = [
+            errorWithInfo.localizedDescription,
+            errorWithInfo.localizedFailureReason,
+            errorWithInfo.localizedRecoverySuggestion
+        ]
+        
+        // Remove optional error messages.
+        let errorMessage = messages.compactMap({ $0 }).joined(separator: "\n")
+        
+        DispatchQueue.main.async { [unowned self] in
+            // Present an alert describing the error.
+            let alertController = UIAlertController(title: "Could not start tracking.", message: errorMessage, preferredStyle: .alert)
+            let restartAction = UIAlertAction(title: "Restart Tracking", style: .default) { _ in
+                alertController.dismiss(animated: true)
```

Comment:
`UIAlertController` automatically dismisses itself when an action is chosen. Were you finding you needed to call this explicitly?

Reply:
Leftover code from long ago...